src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -244,13 +257,20 @@ private void logAcquireLeaseError(String zipFilename, String containerName, Exce
         );
     }
 
-    private boolean readyToAcquireLease(CloudBlockBlob cloudBlockBlob) {
-        String leaseAcquiredAt = cloudBlockBlob.getMetadata().get(LEASE_EXPIRATION_TIME);
-        if (StringUtils.isBlank(leaseAcquiredAt)) {
-            return true;
+    private boolean readyToAcquireLease(CloudBlockBlob cloudBlockBlob, String fileName, String container) {
+        String leaseExpirationTime = cloudBlockBlob.getMetadata().get(LEASE_EXPIRATION_TIME);
+        log.info(
+            "Checking if lease acquired on file {} in container {}. Lease Expiration Time: {}",
+            fileName,
+            container,
+            leaseExpirationTime
+        );
```

Comment:
perhaps we don't need this log entry :wink: . TMI

Reply:
I'm leaving the log for now as it may help us in debugging. I added TODO to cleanup later